judge/runguard.cc

```diff
@@ -101,11 +103,7 @@ const int hard_timelimit = 2;
 const struct timespec killdelay = { 0, 100000000L }; /* 0.1 seconds */
 const struct timespec cg_delete_delay = { 0, 10000000L }; /* 0.01 seconds */
 
-extern int errno;
```

Comment:
Do you want to declare this and external `verbose` from lib.error.

Reply:
Done.